src/contexts/bridge.context.tsx

```diff
@@ -92,8 +108,9 @@ interface BridgeContext {
     bridges: Bridge[];
     total: number;
   }>;
-  bridge: (params: BridgeParams) => Promise<ContractTransaction>;
-  claim: (params: ClaimParams) => Promise<ContractTransaction>;
+  getPendingBridges: (bridges?: Bridge[]) => Promise<PendingBridge[]>;
+  bridge: (params: BridgeParams) => Promise<void>;
+  claim: (params: ClaimParams) => Promise<void>;
```

Comment:
Why convert the return type `ContractTransaction` to `void`?

Reply:
Good catch here, probably a leftover from the first attempts... Fixed!